src/realm/table.hpp

```diff
@@ -1051,6 +1045,102 @@ class ColKeys {
     const Table* m_table;
 };
 
+class LinkChain {
```

Comment:
perhaps a tiny comment about what a LinkChain is?

Reply:
Added a comment